data/variants.txt

```diff
@@ -1362,7 +1412,8 @@ ship "Behemoth" "Behemoth (Hai)"
 	outfits
 		"Sidewinder Missile Launcher" 2
 		"Sidewinder Missile" 100
-		"Pulse Turret" 6
+		"Pulse Turret"
+		"Bullfrog Anti-Missile"
```

Comment:
ya took out 6 turrets & placed only 2 back in

Reply:
Oops.